x-pack/plugins/security_solution/cypress/e2e/entity_analytics/entity_analytics_management_page.cy.ts

```diff
@@ -14,81 +14,144 @@ import {
+      deleteConfiguration();
+      visit(ENTITY_ANALYTICS_MANAGEMENT_URL);
+    });
 
-  beforeEach(() => {
-    login();
-    visitWithoutDateRange(ALERTS_URL);
-    createRule(getNewRule({ query: 'user.name:* or host.name:*', risk_score: 70 }));
-    visit(ENTITY_ANALYTICS_MANAGEMENT_URL);
-  });
+    after(() => {
+      cy.task('esArchiverUnload', 'all_users');
+    });
 
-  after(() => {
-    cy.task('esArchiverUnload', 'all_users');
-  });
+    it('renders page as expected', () => {
+      cy.get(PAGE_TITLE).should('have.text', 'Entity Risk Score');
+    });
 
-  it('renders page as expected', () => {
-    cy.get(PAGE_TITLE).should('have.text', 'Entity Risk Score');
-  });
+    describe('Risk preview', () => {
+      it('risk scores reacts on change in datepicker', () => {
+        const START_DATE = 'Jan 18, 2019 @ 20:33:29.186';
+        const END_DATE = 'Jan 19, 2019 @ 20:33:29.186';
 
-  describe('Risk preview', () => {
-    it('risk scores reacts on change in datepicker', () => {
-      const START_DATE = 'Jan 18, 2019 @ 20:33:29.186';
-      const END_DATE = 'Jan 19, 2019 @ 20:33:29.186';
+        cy.get(HOST_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
+        cy.get(USER_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
 
-      cy.get(HOST_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
-      cy.get(USER_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
+        updateDateRangeInLocalDatePickers(LOCAL_QUERY_BAR_SELECTOR, START_DATE, END_DATE);
 
-      updateDateRangeInLocalDatePickers(LOCAL_QUERY_BAR_SELECTOR, START_DATE, END_DATE);
+        cy.get(HOST_RISK_PREVIEW_TABLE).contains('No items found');
+        cy.get(USER_RISK_PREVIEW_TABLE).contains('No items found');
+      });
 
-      cy.get(HOST_RISK_PREVIEW_TABLE).contains('No items found');
-      cy.get(USER_RISK_PREVIEW_TABLE).contains('No items found');
-    });
+      it('risk scores reacts on change in search bar query', () => {
+        cy.get(HOST_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
+        cy.get(USER_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
 
-    it('risk scores reacts on change in search bar query', () => {
-      cy.get(HOST_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
-      cy.get(USER_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
+        fillLocalSearchBar('host.name: "test-host1"');
+        submitLocalSearch(LOCAL_QUERY_BAR_SELECTOR);
 
-      fillLocalSearchBar('host.name: "test-host1"');
-      submitLocalSearch(LOCAL_QUERY_BAR_SELECTOR);
+        cy.get(HOST_RISK_PREVIEW_TABLE_ROWS).should('have.length', 1);
+        cy.get(HOST_RISK_PREVIEW_TABLE_ROWS).contains('test-host1');
+        cy.get(USER_RISK_PREVIEW_TABLE_ROWS).should('have.length', 1);
+        cy.get(USER_RISK_PREVIEW_TABLE_ROWS).contains('test1');
+      });
+
+      it('show error panel if API returns error and then try to refetch data', () => {
+        cy.intercept('POST', '/internal/risk_score/preview', {
```

Comment:
NIT: Wrapt these intercepts into a task method making clear what you are doing or just add a simple comment about the stubbing you are doing :)